services/grpcService.ts

```diff
@@ -129,7 +129,7 @@ export class GrpcBlockchainClient implements BlockchainClient {
 
       if (transactions.confirmedTransactionsList.length === 0 && transactions.unconfirmedTransactionsList.length === 0) break
 
-      // remove transactions older than the networks
+      // remove transactions older than the network
```

Comment:
Maybe good opportunity to improve or remove this comment as I have 0 clue what it means.

I'm going to approve the PR anyway since this really isn't making it any less confusing than before.

Reply:
@Klakurka  there is a date in the past where we don't look past it when fetching transactions in the past, as if it were the date where that token/network was born. We just ignore any transaction older than that date.

Reply:
I can replace that comment with the longer text of my previous comment if that makes it clearer